### DISCUSSION TITLE
PCB in Canada

### DISCUSSION THREAD

**Discussion by SirDedale** [FundamentalFrequency/.github#85] [General]
Hi all, 
Just ordered 5 PCB for the LMN-3.
If anyone in Canada is interested, I can share those at cost. 

Let me know.

**Comment by beamsville** [comment]
Still got?

**Comment by SirDedale** [comment]
yes sir. ping me on dedale on gmail

**Comment by trash-anger** [comment]
Still got? (👼)

**Reply by SirDedale** [reply]
Yes, same as before ping me and we’ll figure it out.
